src/pages/documentation/iam/tutorials/tabs/messaging-delegate.md

```diff
@@ -278,6 +283,8 @@ public boolean shouldShowMessage(FullscreenMessage fullscreenMessage) {
 
 If you would like to manually integrate the `View` that contains the UI for an in-app message, you can do so by accessing the `WKWebView` directly in a `MessagingDelegate` method.  
 
+IMPORTANT! - The `shouldShowMessage` delegate method is called on a background thread. Any direct interactions with the Message's `WKWebView` made by the delegate should be dispatched back to the main thread.
```

Comment:
I think this would be benefit being in a proper InlineAlert block, so that it would give it a bit more attention.

Reply:
do those work inside of a tab?

Reply:
Oh! My bad. I didn't realize it's in a tab. Then yeah, this functionality is currently not working, so keep it as a note like that for now, but we'll want to push it to an InlineAlert in the future once it _does_ get working.